scripts/run_openstack_kuttl.sh

```diff
@@ -0,0 +1 @@
+
```

Comment:
has this been added by mistake?

Reply:
yes,not sure what happened here, but will remove this